Package.swift

```diff
@@ -1,5 +1,20 @@
+// swift-tools-version: 5.6
```

Comment:
I picked 5.6 here as it's the lowest version we support on the package index. Of course, please feel free to reduce this to whatever you'd like to still support.

The only requirement we have is that `swift package dump-package` runs with the latest stable toolchain.